pyensembl/database.py

```diff
@@ -64,7 +65,7 @@ def _database_indices(self, available_columns):
         indices.append(['feature'])
         return indices
 
-    def _create_database(self):
+    def create_database(self):
         print "Creating database: %s" % self.local_db_path()
```

Comment:
In alliance with Tim's attempt to Python3 all the things, can we wrap this print in parentheses?